src/test/java/emissary/util/ByteUtilTest.java

```diff
@@ -208,4 +209,26 @@ void testGrabLine() {
         assertEquals("This is line three", ByteUtil.grabLine(data, 35), "Last line extraction");
     }
 
+    @Test
+    void testHasnonPrintableValues() {
+        String newLineCarriageTab = "\tThis is line one\r\nThis is line two\nThis is line three\n\nEnding with a tab";
+        assertFalse(ByteUtil.hasNonPrintableValues(newLineCarriageTab.getBytes(StandardCharsets.UTF_8)));
```

Comment:
It's possible there is more than one definition, but I thought non-printing characters were tab, newline, etc.  Maybe I need more concise definition of what we're trying to identify.

Reply:
Also, is it supposed to end with a tab?  \t?

Reply:
Maybe it should be called binary.

Reply:
The previous code allowed tabs, new lines and carriage returns, which makes sense.